docs/examples.rst

```diff
@@ -44,9 +44,9 @@ Basic example
 
 .. code:: php
 
-    <?php echo $this->Form->create('User', ['type' => 'file']); ?>
-    <?php echo $this->Form->input('User.username'); ?>
```

Comment:
Ok -- I've added the old code in comments after each affected code block. Will this work?